scalableminds/webknossos
# [Blocked] Show (restricted) datasets table for non-admins

### URL of deployed dev instance (used for testing):
- https://datasetstableforusers.webknossos.xyz

### Steps to test:
- view datasets table in dashboard as admin: unimported datasets should show as well as "edit" actions
- view datasets table in dashboard as non-admin: only imported datasets should show (without "edit" actions)

### Issues:
- fixes https://discuss.webknossos.org/t/cant-generate-skeleton-tracing-from-dataset-gallery-anymore/989/6

------
